opre_ops/opre_ops/settings/cloudgov.py

```diff
@@ -0,0 +1,27 @@
+"""
+Configuration for running OPRE OPS in cloud.gov.
+"""
+import os
+
+# Import all common settings relevant to both local & cloud:
+from opre_ops.settings.common import *
+
+# https://docs.djangoproject.com/en/3.2/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': os.environ.get('DATABASE_NAME'),
+        'USER': os.environ.get('DATABASE_USER'),
+        'PASSWORD': os.environ.get('DATABASE_PASSWORD'),
+        'HOST': os.environ.get('DATABASE_HOST'),
+        'PORT': '5432'
+    }
+}
+
+# SECURITY: Keep the secret keys used in production secret!
+SECRET_KEY = os.environ.get('SECRET_KEY'),
```

Comment:
Same comment about using [user-provided service credential](https://docs.cloudfoundry.org/devguide/services/user-provided.html)

Reply:
Changed to use a user-provided service credential here!